src/Host/Client/Test/RtvsPackage/BlobsTest.cs

```diff
@@ -175,5 +176,85 @@ public class BlobsTest : IAsyncLifetime {
+        public async Task CompressedBlob() {
+            using (var eval = await _session.BeginEvaluationAsync()) {
+                var createResult = await eval.EvaluateAsync("rtvs:::create_blob(raw(1000000))", REvaluationKind.Normal);
+                createResult.Result.Should().NotBeNull();
+
+                var createCompressedResult = await eval.EvaluateAsync("rtvs:::create_compressed_blob(raw(1000000))", REvaluationKind.Normal);
+                createCompressedResult.Result.Should().NotBeNull();
+
+                var blobId = ((JValue)createResult.Result).Value<ulong>();
+                var expectedData = await eval.EvaluateAsync<byte[]>($"rtvs:::get_blob({blobId})", REvaluationKind.Normal);
+                
+                var blobId2 = ((JValue)createCompressedResult.Result).Value<ulong>();
+                var compressedData = await eval.EvaluateAsync<byte[]>($"rtvs:::get_blob({blobId2})", REvaluationKind.Normal);
+                compressedData.Length.Should().BeLessThan(expectedData.Length);
+
+                byte[] actualData = DecompressData(compressedData);
+                actualData.Should().Equal(expectedData);
+
+                actualData.Should().Equal(expectedData);
+                await eval.ExecuteAsync($"rtvs:::destroy_blob({blobId})");
+                await eval.ExecuteAsync($"rtvs:::destroy_blob({blobId2})");
+            }
+        }
+
+        [Test]
+        public async Task CompressedZeroSizedBlob() {
+            using (var eval = await _session.BeginEvaluationAsync()) {
+                var createResult = await eval.EvaluateAsync("rtvs:::create_blob(raw())", REvaluationKind.Normal);
+                createResult.Result.Should().NotBeNull();
+
+                var createCompressedResult = await eval.EvaluateAsync("rtvs:::create_compressed_blob(raw())", REvaluationKind.Normal);
+                createCompressedResult.Result.Should().NotBeNull();
+
+                var blobId = ((JValue)createResult.Result).Value<ulong>();
+                var expectedData = await eval.EvaluateAsync<byte[]>($"rtvs:::get_blob({blobId})", REvaluationKind.Normal);
+
+                var blobId2 = ((JValue)createCompressedResult.Result).Value<ulong>();
+                var compressedData = await eval.EvaluateAsync<byte[]>($"rtvs:::get_blob({blobId2})", REvaluationKind.Normal);
+
+                byte[] actualData = DecompressData(compressedData);
+                actualData.Should().Equal(expectedData);
+
+                await eval.ExecuteAsync($"rtvs:::destroy_blob({blobId})");
+                await eval.ExecuteAsync($"rtvs:::destroy_blob({blobId2})");
+            }
+        }
+
+        private byte[] DecompressData(byte[] compressedData) {
+            byte[] decompressedData = null;
+            var fs = new FileSystem();
+            string tempFile = fs.GetTempFileName();
+            string dataFile = fs.GetTempFileName();
+            try {
+                fs.FileWriteAllBytes(tempFile, compressedData);
```

Comment:
If you want, you can implement `DeployFilesFixture` that will allow you to store files as test artifacts